src/components/PeoplePage/Parent/Parent.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Person } from '../../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { ParentProps } from '../PeoplePage.types';
+
+function getParentByName(name: string | null, people: Person[]) {
+  const parent = people.find(person => person.name === name);
+
+  return parent || undefined;
+}
+
+export const Parent: FC<ParentProps> = ({ parentName, people }) => {
+  const parent = getParentByName(parentName, people);
```

Comment:
But now we need to do it on every render, let's prepare array with data after we receive it from server to avoid unnecessary iterations